airflow/dag_processing/bundles/git.py

```diff
@@ -127,8 +156,11 @@ def __init__(
             self.log.error("Error creating GitHook: %s", e)
 
     def _initialize(self):
-        self._clone_bare_repo_if_required()
-        self._ensure_version_in_bare_repo()
+        with self.hook.setup_inline_key() as tmp_keyfile:
```

Comment:
You'll also need to do this for refresh.

Reply:
Oops, added it!